cli/command/formatter/cluster_health.go

```diff
@@ -112,6 +130,23 @@ func (c *clusterHealthContext) dpHealthy() bool {
 		c.v.Health.DP.FS.Status == "alivealivealivealive"
 }
 
+func (c *clusterHealthContext) dpDegraded() string {
+	degraded := []string{}
+	if c.v.Health.DP.DirectFSClient.Status != "alive" {
+		degraded = append(degraded, clusterHealthDirectFSClientHeader)
+	}
+	if c.v.Health.DP.Director.Status != "alive" {
+		degraded = append(degraded, clusterHealthDirectorHeader)
+	}
+	if c.v.Health.DP.FSDriver.Status != "alive" {
+		degraded = append(degraded, clusterHealthFSDriverHeader)
+	}
+	if c.v.Health.DP.FS.Status != "alive" {
+		degraded = append(degraded, clusterHealthFSHeader)
+	}
+	return "Degraded (" + strings.Join(degraded, ", ") + ")"
+}
```

Comment:
Let's have table tests for this so that we get to know when we break something unintentionally in the future.